db/schema.rb

```diff
@@ -48,5 +48,36 @@
     t.index ["form_id"], name: "index_pages_on_form_id"
   end
 
+  create_table "published_forms", force: :cascade do |t|
+    t.bigint "form_id"
+    t.json "version"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["form_id"], name: "index_published_forms_on_form_id"
+  end
+
+  create_table "version_associations", force: :cascade do |t|
+    t.integer "version_id"
+    t.string "foreign_key_name", null: false
+    t.integer "foreign_key_id"
+    t.string "foreign_type"
+    t.index ["foreign_key_name", "foreign_key_id", "foreign_type"], name: "index_version_associations_on_foreign_key"
+    t.index ["version_id"], name: "index_version_associations_on_version_id"
+  end
+
+  create_table "versions", force: :cascade do |t|
+    t.string "item_type", null: false
+    t.bigint "item_id", null: false
+    t.string "event", null: false
+    t.string "whodunnit"
+    t.jsonb "object"
+    t.datetime "created_at"
+    t.jsonb "object_changes"
+    t.integer "transaction_id"
+    t.index ["item_type", "item_id"], name: "index_versions_on_item_type_and_item_id"
+    t.index ["transaction_id"], name: "index_versions_on_transaction_id"
+  end
```

Comment:
I can't see migrations for these tables, I assume they are from papertrail and not needed for this PR?

Reply:
💯 this is just because I've been switching between `add-paper-trail` branch and this one and haven't migrated up/down each time.